Gruntfile.js

```diff
@@ -47,7 +47,7 @@ module.exports = function(grunt) {
       // environment variables - see https://github.com/jsoverson/grunt-env for more information
       local: {
         FH_USE_LOCAL_DB: true,
-        FH_SERVICE_MAP: function() {
+        FH_SERVICE_MAP: function () {
```

Comment:
This project was already formatted. This looks like standard formatting profile in Webstorm, however I would not use it as default profile for opensource project. I would recommend to adjust profile or format changes only for blocks of code that were changed (by selecting them).